File: src/main/java/attendance/model/Student.java
package attendance.model;

import lombok.Data;

import javax.persistence.*;

@Data
@Entity
public class Student {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private int roll;

    private Long registrationNumber;

    private String fullName;

    private String email;

    private String department;

    private String semester;
}

File: src/main/java/attendance/model/Student.kt
package attendance.model

import javax.persistence.*

@Entity
data class Student(
        @Id
        @GeneratedValue(strategy = GenerationType.IDENTITY)
        var id: Long,
        var roll: Int,
        var registrationNumber: Long,
        var fullName: String,
        var email: String,
        var department: String,
        var semester: String
)